drivers/SmartThings/matter-switch/src/init.lua

```diff
@@ -154,6 +155,21 @@ local child_device_profile_overrides = {
   { vendor_id = 0x1321, product_id = 0x000D,  child_profile = "switch-binary" },
 }
 
+local fingerprint_profile_overrides = {
+  { vendor_id = 0x1361, product_id = 0x0001 }, -- Inovelli VTM31-SN
+}
+
+local LATEST_CLOCK_SET_TIMESTAMP = "latest_clock_set_timestamp"
+
+local preference_map_inovelli_vtm31sn = {
```

Comment:
It would make sense to move preferences that are specific to a certain device to a sub driver if we are able to. Or these configs could be moved to their own file at least to keep it separate from the base driver where we should try to handle things as generically as possible

Reply:
I moved the inovelli-specific handling to a subdriver in 2e8b4b52da67567d53df71878a9515205ce5b46e